### ISSUE TITLE
Doesn't work in new Azure DevOps

### ISSUE THREAD

**Issue by brownbl1** [ostreifel/vsts-contributions#24]
I'm not sure if this is something that is just misconfigured or if it's broken for some other reason, but I'm getting broken behavior in the latest Azure DevOps. When I try to toggle any of the controls, nothing happens and I get this error in the console:
```
could not parse hash  SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at contributionsHub.tsx:49
    at contributionsHub.tsx:31
    at Object.next (contributionsHub.tsx:12)
    at contributionsHub.tsx:6
    at new Promise (<anonymous>)
    at i (contributionsHub.tsx:2)
    at contributionsHub.tsx:46
    at Array.<anonymous> (contributionsHub.tsx:64)
    at contributionsHub.tsx:31
```

Sometimes it seems to work though, but when I toggle on the changesets option, I get a ton of 404 errors in the network log. 

![image](https://user-images.githubusercontent.com/2136925/50654713-b71bbe00-0f5b-11e9-89c5-c01fae951544.png)

Is this a known issue or something I've done wrong configuring it? 

Thanks for an otherwise fantastic plugin!

**Comment by ostreifel** [comment]
The unexpected end of json input means that you don't have a filter in the url, I suppress that error when the hash is just "".

The changsets 404 are likely b/c you are using the all projects option with out using tfvc for any of them this is expected and should be handled by the extension.

I can you help me understand what you mean by nothing happening?
- No data for current user? Any user?
- freezes when loading - never stops the spinner?
- takes too long to load but loading eventually?

**Comment by brownbl1** [comment]
@ostreifel Thanks for the response. I was specifically looking for changesets, and wasn't seeing any when I thought I should. The toggles did seem to be broken at one point, but after I refreshed the page and specified a different repository, they started to show up. At this point I can't reproduce the behavior I was seeing, and part of it was likely user error. It did seem like it was frozen if I changed to a different user, but that is also working now.

Thanks for a great extension!
